src/Microsoft.DotNet.Archive/ConsoleProgressReport.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.DotNet.Archive
+{
+    public class ConsoleProgressReport : IProgress<ProgressReport>
+    {
+        string currentPhase;
```

Comment:
I promised I wasn't going to 'nit', but this code doesn't conform to our coding standards. This is not a new pattern we want to encourage.